CONTRIBUTING.md

```diff
@@ -59,6 +67,22 @@ need help on it or would like suggestions on how to approach the issue. If so,
 share wildly. If they seem to have a good handle on it, let them work on their
 solution until a challenge comes up.
 
+#### Tips
+
+- At any point you can compare your feature branch to the upstream/main of
+  `LAION-AI/Open-Assistant` but using a URL like this:
+  https://github.com/LAION-AI/Open-Assistant/compare/main...andrewm4894:Open-Assistant:my-example-feature-branch.
+  Obviously just replace `andrewm4894` with your own GitHub user name and
+  `my-example-feature-branch` with whatever you called the feature branch you
+  are working on. This will show the changes that would appear in a PR, so you
```

Comment:
```suggestion
  are working on so something like `https://github.com/LAION-AI/Open-Assistant/compare/main...<your_github_username>:Open-Assistant:<your_branch_name>`. This will show the changes that would appear in a PR, so you
```